dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -544,7 +544,7 @@ HRESULT CFSDropTarget::_DoDrop(IDataObject *pDataObject,
             bLinking = TRUE;
     }
 
-    if (SUCCEEDED(pDataObject->QueryGetData(&fmt)))
+    if (SUCCEEDED(hr = pDataObject->QueryGetData(&fmt)))
```

Comment:
NOTE: OK but this retrieved hr value won't be cared about even in case of non-success.